doc/rbd/rbd-exclusive-locks.rst

```diff
@@ -27,57 +27,55 @@ Exclusive locking is mostly transparent to the user.
    enabled, it obtains an exclusive lock on the image before the first
    write.
 
-#. Whenever any such client process gracefully terminates, it
-   automatically relinquishes the lock.
+#. Whenever any such client process terminates gracefully, the process
+   relinquishes the lock automatically.
 
-#. This subsequently enables another process to acquire the lock, and
-   write to the image.
+#. This graceful termination enables another, subsequent, process to acquire
+   the lock and to write to the image.
 
-Note that it is perfectly possible for two or more concurrently
-running processes to merely open the image, and also to read from
-it. The client acquires the exclusive lock only when attempting to
-write to the image. To disable transparent lock transitions between
-multiple clients, it needs to acquire the lock specifically with
-``RBD_LOCK_MODE_EXCLUSIVE``.
+.. note::
+   It is possible for two or more concurrently running processes to open the
+   image and to read from it. The client acquires the exclusive lock only when
+   attempting to write to the image. To disable transparent lock transitions
+   between multiple clients, the client must acquire the lock with
+   ``RBD_LOCK_MODE_EXCLUSIVE``.
```

Comment:
```suggestion
   ``RBD_LOCK_MODE_EXCLUSIVE`` flag.
```

Reply:
I've made this alteration and prepended a definite article to the whole nounal business that we've introduced with this change.